Streamlined README by relocating and optimizing architecture diagrams. (#142)
1QB-Information-Technologies/ccvm: ccvm_simulators/solvers/README.md

# Solvers

### Solver classes
There are currently two methods available for each solver class [DL-CCVM, MF-CCVM, Langevin] which are **original** (default) and **Adam**.

- `dl_solver.py`: models the delay line coherent continuous-variable machine (DL-CCVM).

> - `mf_solver.py`: models the measurement-feedback continuous-variable machine (MF-CCVM).

Reviewer: ```suggestion
- `mf_solver.py`: models the measurement-feedback coherent continuous-variable machine (MF-CCVM).
```

> - `langevin_solver.py`: models typical Langevin dynamics as a system of SDE (Langevin).

Reviewer: ```suggestion
- `langevin_solver.py`: models typical Langevin dynamics as a system of stochastic differential equations (Langevin).
```

Reviewer 2: On line 7, you are missing the word "coherent".

On line 6, you are missing the hyphen in "delay-line".

Could we please remove the "s" from "(SDEs") so it just reads "(SDE)"? 
When _introducing_ an abbreviation, the abbreviation is not pluralized, even if the term before it is in plural form; after that, each subsequent use of the abbreviation is pluralized if referring to the plural form of the term.

Author: Thanks for all the feedback. The comments have been addressed.

Reviewer 3: Thanks Marko, it should be `SDE`

#### The solver hierarchy 
The diagram portrays the relationships among different solvers and their

> association with the abstracted CCVM solver class.

Reviewer: ```suggestion
association with the abstract CCVM solver class.
```
